docker/package/tezos_baking_wizard.py

```diff
@@ -468,7 +468,16 @@ def bootstrap_node(self):
         self.import_snapshot()
 
         self.systemctl_start_action("node")
-        proc_call("sleep 5")
+
+        status_code = None
+        while status_code != 200:
+            try:
+                status_code = urllib.request.urlopen(
+                    "http://localhost:8732/chains/main/blocks/head"
```

Comment:
Perhaps, it's worth getting the node RPC address from the service configuration instead of hardcoding it here. It's defined as `Environment="NODE_RPC_ADDR=127.0.0.1:8732"` in the `tezos-node-<network>` services

Reply:
One more nitpick about this: the RPC endpoint used is protocol-independent and doesn't rely on much (only on the existence of the `main` chain_id`), but why not use a simpler one?

How about [`/version`](http://tezos.gitlab.io/shell/rpc.html#get-version)?